src/web/mjs/connectors/manmanapp.mjs

```diff
@@ -0,0 +1,79 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ManmanApp extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'manmanapp';
+        super.label = 'Manman Comic 漫漫漫画';
+        this.tags = [ 'manga', 'webtoon', 'chineses' ];
+        this.url = 'https://www.manmanapp.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'li.title');
+        console.log(data)
```

Comment:
Remove debug info from productive code, also prefer chrome debugger over console logging

Reply:
ik how to use console log 
idk how to use chrome debugger properly

Reply:
1. in HakuNeko press F12
2. switch to sources
3. navigate to the file you want to debug and open it
4. set a breakpoint at the beginning of the line
5. interact with the application, e.g. select manga/chapter
6. wait to reach breakpoint
7. hover cursor over variables to see values or use console.log to dump data
8. use the debugger controls such as step-in, step-over, continue to proceed

See also: https://developers.google.com/web/tools/chrome-devtools/javascript

![image](https://user-images.githubusercontent.com/27136773/98365460-c033b580-2032-11eb-995b-68b6df345eab.png)

Reply:
ty